omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java

```diff
@@ -230,6 +230,7 @@ private String getRedirectUrl(PageRequest pageRequest) {
 	public String post(@RequestParam(value = "username", required = false) String username,
 	                   @RequestParam(value = "password", required = false) String password,
 	                   @RequestParam(value = "sessionLocation", required = false) Integer sessionLocationId,
+					   @RequestParam(value = "client-timezone", required = false) String clientTimezone,
```

Comment:
Is there any reason why you used `client-timezone` instead of `clientTimezone`?

Reply:
No specific reason, i will change it to `clientTimezone`